blog.md

```diff
@@ -14,28 +14,28 @@ while also promoting creation of small PRs &mdash;
 **a git repository organizer, and a rebase/merge/push/pull automation tool, named Git Machete.**
 
 It allows for looking at the state of the repository from a previously inexperienceable perspective.
-It puts the primary focus on branches (rather than commits), their relationship to each other,
-and their relationship to the remote repository.
-Moreover, it provides both diagnostic and corrective functionalities
+It puts the primary focus on branches (rather than commits),
+their relationship to each other and also to the remote repository.
+Moreover, it gives both diagnostic and restorative functionalities
```

Comment:
Hmmm this sentence sounds a bit too formal/hazy... let's just skip it completely TBH